bigquery-connector-common/pom.xml

```diff
@@ -128,6 +128,11 @@
       <artifactId>gson</artifactId>
       <version>2.9.1</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.beam</groupId>
+      <artifactId>beam-sdks-java-core</artifactId>
+      <version>2.43.0</version>
```

Comment:
This version has [2 CVEs](https://mvnrepository.com/artifact/org.apache.beam/beam-sdks-java-core/2.43.0), latest (2.46.0) is ok.

Also, the jar is 5MB, do we need it all for just one class? As an alternative - perhaps we can trim the redundant classes using the filter option in the shade plugin